src/getCoinCombination.js

```diff
@@ -7,15 +7,15 @@
  */
 function getCoinCombination(cents) {
   let currentAmount = cents;
-  const values = [1, 5, 10, 25];
+  const values = [25, 10, 5, 1]; // Починаємо з кварталів
```

Comment:
The comment is in Ukrainian. Consider translating it to English for consistency and better understanding by a wider audience.